doc/Sphinx/post-processing.rst

```diff
@@ -98,6 +98,8 @@ Open a Scalar diagnostic
   * ``units``: A unit specification (see :ref:`units`)
   * ``data_log``:
      | If ``True``, then :math:`\log_{10}` is applied to the output.
+  * ``data_transform``:
+     | If ``callable``, is applied to the output.
```

Comment:
Using the word "callable" is a little technical. I always try to make the documentation as accessible as possible. Wouldn't it be better to say something like "a function applied to the output if defined" ?

Reply:
Good point - I'll change the documentation here to make it a bit easier to understand